public/app/features/search/types.ts

```diff
@@ -41,6 +41,8 @@ export interface DashboardSectionItem {
   uid?: string;
   uri: string;
   url: string;
+  sortMeta?: number;
```

Comment:
```suggestion
  sortMeta: number;
```
If we remove the `omitempty` in the .go file, this will always have a value.

Reply:
This type is used outside of Search as well, so having it non-optional will break other use-cases.